.github/workflows/ci-build.yaml

```diff
@@ -311,7 +311,7 @@ jobs:
           file: coverage.out
       - name: Perform static code analysis using SonarCloud
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN || secrets.GITHUB_TOKEN }}
```

Comment:
Probably not this one.  :-)

Reply:
@crenshaw-dev : is it possible u to do change and merge this or may be will change post merge

Reply:
```suggestion
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```